voidcrew/code/modules/surgery/tools.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/oil_cleanser
+	name = "\improper oil cleanser"
+	desc = "Sometimes those IPCs need an oil change."
+
+	// fuck you im not spriting
+	icon = 'icons/obj/syringe.dmi'
+	icon_state = "maintenance"
+	item_state = "medipen"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+
+	item_flags = SURGICAL_TOOL
+	force = 10
```

Comment:
does this really have to hit like a truck

Reply:
copied from scalpel i forgot to change it >:)